starter-code/functions-and-arrays.js

```diff
@@ -63,7 +88,14 @@ console.log(averageLength);
 
 // Unique Arrays
 function uniquifyArray (array) {
```

Comment:
This is an original solution. It uses stuff we haven't seen yet. I like it but for the sake of it you could try a simpler one.